packages/app-builder-lib/templates/nsis/installer.nsi

```diff
@@ -49,6 +43,12 @@ Function .onInit
     !insertmacro preInit
   !endif
 
+  !ifdef ONE_CLICK
+    !include "oneClick.nsh"
+  !else
+    !include "assistedInstaller.nsh"
+  !endif
+
```

Comment:
@beyondkmp can you please review this change? I'm not familiar enough with Windows nsis scripting to be comfortable reviewing this